.github/workflows/dotnet-core.yml

```diff
@@ -0,0 +1,25 @@
+name: .NET Core
```

Comment:
Let's call the pipeline, and file, `CI`.

Reply:
Pipeline and filename now updated